src/chocolatey.tests/infrastructure.app/commands/ChocolateyPinCommandSpecs.cs

```diff
@@ -412,7 +414,7 @@ public void should_call_nuget_service_list_run_when_command_is_list()
                 configuration.PinCommand.Command = PinCommandType.list;
                 command.run(configuration);
 
-                nugetService.Verify(n => n.list_run(It.IsAny<ChocolateyConfiguration>(), false), Times.Once);
+                nugetService.Verify(n => n.list_run(It.IsAny<ChocolateyConfiguration>(), true), Times.Once);

```

Comment:
This one I actually know, now.  After the changes to make things stream output, I had to set this to TRUE so the results would get enumerated (and the method under test would get called).
Of course, it goes away in the next commit, because we got rid of the boolean and moved it to the super-all-knowing-config-object.